### PR TITLE
AUT-192: Add 'How to increase disk space' page

### PR DESCRIPTION
**Pull request by adityapahuja**
This change adds a new page about 'How to increase disk space in Carrenza or UKCloud'.

Author: @adityapahuja

### CODE REVIEW

File: source/documentation/how-to-increase-disk-space.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+    I/O size (minimum/optimal): 512 bytes / 512 bytes
+    Disk identifier: 0x00000000
+    
+    Disk /dev/sdd doesn't contain a valid partition table
+    ```
+
+15. Run the following command to resize the partition.
+
+    ```
+    $ resize2fs /dev/sdd1
```
Comment (blairboy362) [inline source/documentation/how-to-increase-disk-space.md]: I'm assuming we're using ext4 (or ext3) everywhere but we never confirm that, as far as I can see. I'm unfamiliar with Verify's infrastructure so I've no idea how likely it is we're using different filesystem types.

File: source/documentation/how-to-increase-disk-space.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+8. Click on the cog.
+9. Click on properties.
+10. In the new tab, click on hardware.
+11. Add an extra disk.
+
+    | IMPORTANT: Make sure you add a paravirtual SCSI. Other disks can prevent the virtual machine from booting up. |
+    | --- |
+
+12. Select a relevant size for the volume.
+13. Check that the volume we are expanding.
```
[the remark this answers is not in the document]
Comment (adityapahuja) [reply source/documentation/how-to-increase-disk-space.md]: I copied this line from 'How to increase disk spaces in boxes' page in [Verify Puppet Wiki](https://github.com/alphagov/verify-puppet/wiki/Increase-disk-space-on-boxes). I think it means that we need to check the current size of volume before expanding it.

File: source/documentation/how-to-increase-disk-space.md (hunk 1 above)
Comment (adityapahuja) [reply source/documentation/how-to-increase-disk-space.md]: The details taken from one server running in Carrenza are as follows.

```
/dev/sda:  x86 boot sector
/dev/sda1: Linux rev 1.0 ext4 filesystem data, UUID=... (needs journal recovery) (extents) (large files) (huge files)
/dev/sdb:  LVM2 PV (Linux Logical Volume Manager), UUID: ..., size: 107375230976
```
The root volume (/dev/sda1) is using ext4. Non-root volume is using LVM2 PV.

File: source/documentation/how-to-increase-disk-space.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (adityapahuja) [reply source/documentation/how-to-increase-disk-space.md]: I have fixed it.

**Pull request merged**